### PR TITLE
Magic sudo/front end

### PR DESCRIPTION
**Pull request by magic-sudo**
Added nanoc and removed flask app.

### CODE REVIEW

File: Rules [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#
+# * Item identifiers start and end with a slash (e.g. “/about/” for the file
+#   “content/about.html”). To select all children, grandchildren, … of an
+#   item, use the pattern “/about/*/”; “/about/*” will also select the parent,
+#   because “*” matches zero or more characters.
+
+compile '*' do
+  if item[:extension] == 'md'
+    filter :kramdown
+    layout 'default'
```
Comment (pepol) [inline Rules]: Do we really want to have 2 different layouts for `markdown -> default` vs `everything else -> page`?

File: layouts/page.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Home"
+---
```
Comment (pepol) [inline layouts/page.html]: I don't think this should be here, in layout.

File: output/index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (pepol) [inline output/index.html]: Please remove this file. Generated files have absolutely **NO** right being in the source.

**Comment by pepol** [comment]
@magic-sudo thanks, but please address my comments.

Addiotionally, please remove all files within `tmp/` directory, as well as `output/` directory. If some files in the output directory are considered static, just move them (within their specific directory) into root (that is, `output/style/style.css` becomes `style/style.css`. Compilation rules deal with those files without any problem.

**Comment by magic-sudo** [comment]
All done.

**Comment by magic-sudo** [comment]
Done.

**Comment by pepol** [comment]
@magic-sudo LGTM, please change commit message (for example, remove any mention of flask, since we don't do anything with flask here), also change component to `web`.

**Comment by magic-sudo** [comment]
what component ?

**Comment by pepol** [comment]
Commit message consits of:

 - summary line
 - (optional) long description
 - tag lines

Summary line is up to 52 characters long and is usually in the form: `component: short description`

HTH ;-)

**Comment by magic-sudo** [comment]
done

**Comment by pepol** [comment]
@magic-sudo thanks! Merging.

**Pull request merged**